packages/lib/src/components/ThreeDS2/components/Challenge/types.ts

```diff
@@ -7,6 +7,7 @@ export interface DoChallenge3DS2Props extends ChallengeData {
     onCompleteChallenge: (resolveObject: ThreeDS2FlowObject) => void;
     onErrorChallenge: (rejectObject: ThreeDS2FlowObject) => void;
     onActionHandled: (rtnObj: ActionHandledReturnObject) => void;
+    onFormSubmit: (w) => void;
```

Comment:
Could we make the type and name of `w` more specific, i suppose it's an analytic message and the type is `string`?

I see it's being defined in many places, maybe we can create a share `type` for it.

Reply:
Done